products/vuetify.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Vuetify
+category: framework
+iconSlug: vuetify
+permalink: /vuetify
+alternate_urls:
+-   /vuetify
+
+versionCommand: npm list vuetify
+releasePolicyLink: https://vuetifyjs.com/en/introduction/roadmap/#released
+activeSupportColumn: true
+releaseDateColumn: true
+
+auto:
+-   npm: vuetify
+
```

Comment:
The [Jupyter/Python variants](https://repology.org/projects/?search=vuetify) don't seem to use the same versioning so ignoring them.

```suggestion

identifiers:
-   purl: pkg:npm/vuetify
```